index.html

```diff
@@ -0,0 +1,146 @@
+  <title>CC Global Summit Memorial</title>
+</head>
+<body>
+  <header>
+    <nav class="navmenu">
+    <div class="container">
+      <div class="btn-x">
+      <a href="#"><img src="./images/Close.png" alt=""></a>
+      </div>
+        <ul>
+          <li class="nav-mobile"><a href="about.html">About</a></li>
+            <li class="nav-mobile"><a href="">Program</a></li>
+            <li class="nav-mobile"><a href="">Events</a></li>
+            <li class="nav-mobile"><a href="">Sponsor</a></li>
+            <li class="nav-mobile"><a href="">News</a></li>
+            <!-- <li class="nav-mobile"><a href="#aboutme_sec">About</a></li> -->
+        </ul>
+      </div>
+      </nav>
+
+    <nav class="fast-nav">
+      <div class="social">
+        <ul>
+          <li><a href=""><img src="./images/Github1.png" alt=""></a></li>
+          <li><a href=""><img src="./images/Twitter1.png" alt=""></a></li>
+          <li><a href="">English</a></li>
+          <li><a href="">My Page</a></li>
+          <li><a href="">Logo out</a></li>
+        </ul>
+      </div>
+    </nav>
+    <nav class="item-bar">
+      <div class="nav-last">
+          <ul>
+            <li><a href="" class="nav-last_img"><img src="./images/logo-first.png" alt=""></a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="">Program</a></li>
+            <li><a href="">Events</a></li>
+            <li><a href="">Sponsor</a></li>
+            <li><a href="">News</a></li>
+            <li><a href="" class="Fbutton">Gallery</a></li>
+          </ul>
+      </div>
+    </nav>
+  </header>
+  <section class="section1">
+    <div class="main-text">
+      <div class="sub-text">
+        <div class="toplogo">
+        <span class="ham-btn"><img src="./images/mobile_menu.png" alt=""></span>
+        <a href="#home" class="toplogo1"><img src="./images/logo-first.png" alt=""></a>
+      </div>
+      <p class="welcome" id="home">"Hello Gallery!"</p>
+      <h1 class="main-title">CREATIVE COMMON<br>GLOBAL SUMMITS <br> 2022</h1>
+      <div class="saying">
+        <p class="paragragh1">“Make your lives a masterpiece, you only get one canvas.” “The art of life is to know how to enjoy a little and to endure much.” “In the history of a soul's evolution there is a critical point of the human incarnation that decides for us whether we stay there, go down or progress upwards.</p>
+      </div>
+    </div>
+      <footer>
+        <h2>2022.03.07(Mon)~12(Fri)</h2>
+        <p>@National Museum of Korea, Art Center Nabi</p>
+      </footer>
+    </div>
+  </section>
+  <section class="main-programs">
+    <h2>Main Program</h2>
+    <hr class="fast-line">
+    <div class="program-parts">
+      <div class="part">
+        <img src="./images/lecture.png" alt="">
+        <h3>Lecture</h3>
+        <p>Listen the amazing stories about Capablanca's career</p>
+      </div>
+      <div class="part">
+        <img src="./images/exhibition.png" alt="">
```

Comment:
- [**OPTIONAL** ] I suggest giving the actual values of the `alt` attribute instead of an empty string for accessibility reasons.